src/amo/pages/UserProfileEdit/index.js

```diff
@@ -595,6 +604,24 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                 onChange={this.onFieldChange}
                 value={this.state.displayName}
               />
+              {isReviewer && user && user.reviewer_name !== undefined && (
+                <div>
```

Comment:
I think we should have a `<React.Fragment>` here (instead of using a `<div>`).

Reply:
978850dcd00fa09eb4c53da74330fcd373eaa5b1